src/spaced/spaced/game/damage.hpp

```diff
@@ -1,17 +1,27 @@
 #pragma once
-#include <bave/core/time.hpp>
-#include <glm/vec2.hpp>
+#include <bave/imgui/im_text.hpp>
+#include <bave/app.hpp>
 
 namespace spaced {
 class Damage {
```

Comment:
Hmm, I think we should pivot to calling this `Healt`h or `HitPoints` instead, since that's what it has emerged to represent. 😄